public/app/features/alerting/unified/components/rules/RuleDetails.tsx

```diff
@@ -53,6 +55,38 @@ export const RuleDetails: FC<Props> = ({ rule }) => {
   );
 };
 
+interface EvaluationBehaviorSummaryProps {
+  rule: CombinedRule;
+}
+
+const EvaluationBehaviorSummary = ({ rule }: EvaluationBehaviorSummaryProps) => {
+  let forDuration: string | undefined;
+  let every: string | undefined;
+
+  if (isGrafanaRulerRule(rule.rulerRule)) {
+    forDuration = rule.rulerRule.for;
+    every = rule.group.interval;
+  } else if (isAlertingRulerRule(rule.rulerRule) && rule.rulerRule.for) {
+    forDuration = rule.rulerRule.for;
+    every = rule.group.interval;
+  }
+
+  return (
+    <>
+      {every && (
+        <DetailsField label="Evaluate" horizontal={true}>
+          Every {every}
+        </DetailsField>
+      )}
+      {forDuration && (
+        <DetailsField label="For" horizontal={true}>
+          {forDuration}
+        </DetailsField>
```

Comment:
Should we also render the warning if the evaluation interval global limit is exceeded? :)

Reply:
I think we're already showing it in the alert rule list, right? I'm not sure if it makes sense to display it here again since the details view is also shown when a row is un-collapsed – we'd show it twice I think

Reply:
Yep, you're right!